cluster/gce/gci/configure-helper.sh

```diff
@@ -921,8 +962,8 @@ function start-kubelet {
   if [[ -n "${ENABLE_CUSTOM_METRICS:-}" ]]; then
     flags+=" --enable-custom-metrics=${ENABLE_CUSTOM_METRICS}"
   fi
-  if [[ -n "${NODE_LABELS:-}" ]]; then
-    flags+=" --node-labels=${NODE_LABELS}"
+  if [[ -n "${LOCAL_NODE_LABELS:-}" ]]; then
+    flags+=" --node-labels=${LOCAL_NODE_LABELS}"
```

Comment:
I see why you can't use config-test.sh. I think it would be cleaner to drop LOCAL_NODE_LABELS as a global variable and drop  prepare-node-labels. Just deal with this here. I think this would be cleaner:

```sh
local node_labels=""
if [[ "${KUBERNETES_MASTER:-}" != "true" && "${KUBE_PROXY_DAEMONSET:-}" == "true" ]]; then
  node_labels="beta.kubernetes.io/kube-proxy-ds-ready=true"
fi
if [[ -n "${NODE_LABELS:-}" ]]; then
  local node_labels="${node_labels:+${node_labels},}${NODE_LABELS}"
fi
if [[ -n "${node_labels:-}" ]]; then
  flags+=" --node-labels=${node_labels}"
fi

Reply:
Thanks, this looks neat, applied. PTAL

Reply:
I mean't to say that this should go in kubelet right where we set flags+=" --node-labels..." today

Reply:
Sorry, missed this comment, on it...